primitives/src/transaction.rs

```diff
@@ -392,7 +392,7 @@ impl OutPoint {
 impl fmt::Display for OutPoint {
     #[inline]
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        write!(f, "{}:{}", self.txid, self.vout)
+        write!(f, "{:?}:{}", self.txid, self.vout)
```

Comment:
I really, really, really don't like this conceptually. It means that we're leaking debug representation in public API even though *currently* it's the same, it could change merely by updating the `hashes` crate (which doesn't require the test in this crate to run) and even if we decide to stabilize `Debug` impl in `hashes`, that seems like unnecessary restriction. Why not just cfg-out the entire impl?